syntax = "proto3";

package state;

message AppState {
    repeated string user_progress = 1;
}